tech-specs/2026-06-codegen/presentation/src/index.css

```
@@ -0,0 +1,221 @@
+}
+
+@utility pulse-dot {
+  animation: pulse-dot 1.6s ease-out infinite;
+}
+
+@keyframes blink {
+  0%, 49% { opacity: 1; }
+  50%, 100% { opacity: 0; }
+}
+
+@utility blink {
```

Stylelint (17.13.0)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)